packages/server/src/koaJsonRpc/index.ts

```diff
@@ -128,16 +144,22 @@ export default class KoaJsonRpc {
         if (e instanceof InvalidParamsError) {
           ctx.body = jsonResp(body.id, new InvalidParamsError(e.message), undefined);
           ctx.status = 400;
+          ms = Date.now() - this.startTimestamp;
+          this.logger.error(`${messagePrefix} ${ctx.status} (INVALID PARAMS ERROR) ${ms} ms`);
```

Comment:
We've added error and warning case. 
I think we're missing bringing the info case for successful calls that would produce a 200.
Might be good to have all of the logs in one file

Reply:
The case for successful calls that produce a 200 is logged in the server.ts in the logAndHandleResponse function.  The format for the message logged was taken from that.

Reply:
Per offline convo, seems like having all errors in this file is okay with success in `server.ts`